packages/neuron-wallet/src/models/system-script-info.ts

```diff
@@ -35,6 +35,10 @@ export default class SystemScriptInfo {
 
   private multiSignOutPointInfo = new Map<string, OutPoint>()
 
+  public getDaoScriptHash(): Hash {
+    return utils.computeScriptHash(SystemScriptInfo.DAO_SCRIPT)
```

Comment:
the `DAO_SCRIPT` is static, why we should use a function to compute it instead of just export it as a static. Is there any context of `static DAO_SCRIPT_HASH = utils.computeScriptHash(DAO_SCRIPT) is not a good pattern and somehow doesn't work`

Reply:
Can you help me be by fetching this branch on your local machine and change to `static DAO_SCRIPT_HASH = utils.computeScriptHash(DAO_SCRIPT)`, I thought it should work, but it just doesn't, always prompting an error when i ran `yarn test`, didn't find a proper reason but class method works, so I just blamed it on `static` methods.